gef.py

```diff
@@ -8788,30 +8788,30 @@ class DereferenceCommand(GenericCommand):
     command."""
 
     _cmdline_ = "dereference"
-    _syntax_  = "{:s} [LOCATION] [l[NB]]".format(_cmdline_)
+    _syntax_  = "{:s} [LOCATION] [[l]NB] [rLOCATION]".format(_cmdline_)
```

Comment:
new syntax: `r` prefix plus `LOCATION`

sidenote: I fixed `[l[NB]]` to `[[l]]NB]` because the `l` is optional not the `NB`

Reply:
If `NB` isn't optional then it should be `[l]NB` not `[[l]NB]`

Reply:
Sorry, I wasn't precise enough: omitting  it, having just NB and having lNB is fine. So in my book `[[l]NB` is correct.

Reply:
No sweat, I just forgot whether NB was optional (until I reread the code)